src/data/builder-tools.js

```diff
@@ -762,6 +762,14 @@ export const Showcases = [
     getstarted: null,
     tags: ["operatortool", "cli"],
   },
+  {
+    title: "Plutip",
+    description: "Cardano tool that aims to help dApp developers with integration testing and contracts debugging using disposable private network",
+    preview: require("./builder-tools/plutip.png"),
+    website: "https://github.com/mlabs-haskell/plutip",
+    getstarted: "https://github.com/mlabs-haskell/plutip/blob/master/README.md#tutorials",
+    tags: ["haskell", "testing", "private network"],
```

Comment:
```suggestion
    tags: ["haskell", "testing"],
```
Using undefined tags doesn't create them automatically: they have to be deliberately added.  One way of resolving this would be to add a `privatenetwork` tag (our convention doesn't use spaces or hyphens) but that would be done by reviewers & by consensus.  Until if & when that happens the simplest thing is just to drop the tag.

Reply:
Thank you for review.
Tag dropped.